lib/koans/11_structs.ex

```diff
@@ -59,4 +59,14 @@ defmodule Structs do
     airline = %Airline{plane: %Plane{maker: :boeing, passengers: 200}}
     assert update_in(airline.plane.passengers, &(&1 + 2)) == ___
   end
+
+  koan "Use the put_in macro with atoms to replace a nested value in a non-struct" do
+    airline = %{
```

Comment:
K, I will put it all on one line. I make things multiline so I can see teh brace matches easier.